src/component/huskelapp/redigering/huskelapp-redigere-modal.tsx

```diff
@@ -111,9 +112,16 @@ function HuskelappRedigereModal() {
                 brukerFnr: brukerFnr,
                 enhetId: enhetId
             })
-                .then(() => fetchHuskelapp(brukerFnr.toString(), enhetId ?? ''))
-                .then(res => res.data)
-                .then(setHuskelapp)
+                .then(() =>
+                    setHuskelapp({
```

Comment:
Bare for å forstå: siden dette er en mutate, så vil feltene under erstattes med riktig data når vi får det fra backenden?

Reply:
jupp!
Egentlig så hadde jeg kun () => setHuskelapp() her for å få hentet opp igjen den huskelappen som ble lagret (med id sov)... altså kun erstatte 
```
.then(() => fetchHuskelapp(brukerFnr.toString(), enhetId ?? ''))
                .then(res => res.data)
                .then(setHuskelapp)
```

men så tenkte jeg at vi kan jo sette inn midlertidig (mens swr henter ny data), tilfelle det kan hjelpe på problemet veilederne har ved at de opplever at ting blir borte etter opprettelse. men ja, det vil være midlertidig, frem til swr har fått gjort et nytt kall til bakcenden 👍